FlowCrypt/src/main/java/com/flowcrypt/email/database/dao/RecipientDao.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * © 2016-present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com
+ * Contributors: DenBond7
+ */
+
+package com.flowcrypt.email.database.dao
+
+import android.database.Cursor
+import androidx.lifecycle.LiveData
+import androidx.room.Dao
+import androidx.room.Query
+import androidx.room.Transaction
+import com.flowcrypt.email.database.entity.RecipientEntity
+import com.flowcrypt.email.database.entity.relation.RecipientWithPubKeys
+import kotlinx.coroutines.flow.Flow
+
+/**
+ * This object describes a logic of work with [RecipientEntity].
+ *
+ * @author DenBond7
+ * Date: 17.05.2017
+ * Time: 12:22
+ * E-mail: DenBond7@gmail.com
+ */
+@Dao
+interface RecipientDao : BaseDao<RecipientEntity> {
+  @Query("SELECT * FROM recipients")
+  suspend fun getAllRecipients(): List<RecipientEntity>
+
+  @Query("SELECT * FROM recipients")
+  fun getAllRecipientsLD(): LiveData<List<RecipientEntity>>
+
+  @Query("SELECT recipients.* FROM recipients INNER JOIN public_keys ON recipients.email = public_keys.recipient GROUP BY recipients.email ORDER BY recipients._id")
```

Comment:
I can do that. But in that case, this line will not be useful for me. It refers to `Android Studio`. When I have a single line, `Android Studio` does checks and shows me warnings if I do some error in SQL syntax. 

These long lines are nervous me too.

Reply:
You can try use kotlin mult-line string synatax, i.e. 
```
"""
some multi-line
SQL
"""
```

Reply:
unfortunately, it doesn't work
![image](https://user-images.githubusercontent.com/2863246/143429350-caaad432-49fc-487b-8514-30bb3d6fdd9a.png)

Reply:
Hm. I guess in this case it makes sense to have an exception, since AndroidStudio can't syntax-highlight multiple line SQL?

@DenBond7 please see which plugin is responsible for this, make sure you are on the latest one, and potentially submit a bug report to their github or other place. Or find existing bug report to mention that this is affecting us. It seems like quite an oversight.